src/core/ext/transport/cronet/transport/cronet_transport.cc

```diff
@@ -624,14 +653,21 @@ static void on_response_trailers_received(
   for (size_t i = 0; i < trailers->count; i++) {
     CRONET_LOG(GPR_DEBUG, "trailer key=%s, value=%s", trailers->headers[i].key,
                trailers->headers[i].value);
+    grpc_slice key = grpc_slice_intern(
+        grpc_slice_from_static_string(trailers->headers[i].key));
+    grpc_slice value;
+    if (grpc_is_binary_header(key)) {
```

Comment:
This code seems duplicated with header processing. Create a separate function and use in both places?